src/library_webgl.js

```diff
@@ -8,9 +8,23 @@ var LibraryGL = {
 
   // For functions such as glDrawBuffers, glInvalidateFramebuffer and glInvalidateSubFramebuffer that need to pass a short array to the WebGL API,
   // create a set of short fixed-length arrays to avoid having to generate any garbage when calling those functions.
-  _tempFixedLengthArray__postset: 'for (var i = 0; i < 32; i++) __tempFixedLengthArray.push(new Array(i));',
+  _tempFixedLengthArray__postset: 'for (var ____n = 0; ____n < 32; ++____n) __tempFixedLengthArray.push(new Array(____n));',
   _tempFixedLengthArray: [],
 
+  _miniTempFloatBuffers: [],
+  _miniTempFloatBuffers__postset:
+      'var __floatBuf = new Float32Array(' + {{{ GL_POOL_TEMP_BUFFERS_SIZE }}} + ');\n'
```

Comment:
how about `glFloatBuf`?

I don't think we need `__` prefixing for things like this, we don't do that for other things. We just use natural JS names.

And I think mentioning in the name that it is for GL would be clearer.

Reply:
probably not `glFloatBuf` though as it sounds too much like a real thing. Maybe `tempGLFloatBuf`?

Reply:
If a non-underscored name is used, that might conflict with a real name that user brings from external code?

I think we used to apply underscores for internal things that are not exposed to users to access?

Reply:
I don't think we've had a consistent policy on prefixing of our internal things. Maybe that's in part to it not always being clear if something is internal or not.

But if it's clear this is not for users, then `__tempGLFloatBuf` sounds ok to me.

In theory the cleaner thing for this is to use a closure, but that would increase code size sadly.

Reply:
Renamed to `__temp_webgl_float_buf` and `__temp_webgl_int_buf`.